nilearn/maskers/tests/test_multi_nifti_labels_masker.py

```diff
@@ -108,8 +108,6 @@ def test_multi_nifti_labels_masker():
 
     # check exception when transform() called without prior fit()
     masker11 = MultiNiftiLabelsMasker(labels11_img, resampling_target=None)
-    with pytest.raises(ValueError, match="has not been fitted. "):
-        masker11.transform(fmri11_img)
```

Comment:
Is this already being tested somewhere else? same thing with other maskers

Reply:
yes 
as long as you run nilearnn._utils.class_inspect.check_estimator on your estimator in the tests (which is the case for now) then I have added a `check_masker_fitted` in `class_inspect`.

Reply:
see https://github.com/nilearn/nilearn/pull/5029/files#r1917136093

Reply:
The idea is to have a single 'standardized' check that we run on all estiamtors that have the tag masker.

Reply:
Cool!